packages/grid/src/vaadin-grid-keyboard-navigation-mixin.js

```diff
@@ -614,6 +614,9 @@ export const KeyboardNavigationMixin = (superClass) =>
     _onTabKeyDown(e) {
       const focusTarget = this._predictFocusStepTarget(e.composedPath()[0], e.shiftKey ? -1 : 1);
 
+      // Prevent focus-trap logic from intercepting the event.
+      e.stopPropagation();
+
```

Comment:
Can a test be added that the following use case still works afterwards? 

```java
Grid<MyObject> grid;

grid.addComponentColumn(this::createActionRenderer);

private Component createActionRenderer(MyObject) {
   var layout = new HorizontalLayout();
   layout.add(new Button("Action 1", e -> {});
   layout.add(new Button("Action 2", e -> {});
   return layout;
}
```

Focusing the grid and pressing `ENTER` on the component column allows to use `TAB` to access the buttons by keyboard (which is essential for keyboard accessibility).

Reply:
Thanks for looking into it! (I'm not a front end expert)